Formula/fastme.rb

```diff
@@ -0,0 +1,31 @@
+class Fastme < Formula
+  desc "Accurate and fast distance-based phylogeny inference program"
+  homepage "http://www.atgc-montpellier.fr/fastme/"
+  url "https://gite.lirmm.fr/atgc/FastME/raw/master/tarball/fastme-2.1.5.2.tar.gz"
+  sha256 "834e1f555b6057b23650cb214c1f2f899978d72d085fca863fb3f03f2bbf00fc"
+
+  depends_on "gcc"
+  fails_with :clang
+
+  def install
+    system "./configure", "CC=#{ENV.cc}",
```

Comment:
`"CC=#{ENV.cc}"` should be unnecessary

Reply:
Whoops!

It appears that a new version was released in the mean time. The repo owner then removed the older version of the tarball. I was previously using the URL that did not explicitly reference the commit tag of the release, but the master branch and as such it disappeared.

I've updated the URL to refer explicitly to the commit tag with the correct tarball.

Also, I've updated the formula for the new version. This now means that the current commit message (2.1.5.2) is not correct, but it should be 2.1.6. I was unsure how you wanted this handled, since the guide says all commits should be squashed. Let me know what the best way is to do this.

Reply:
- You can reword the commit message when squashing (use the `reword` option).
- Can you please try to remove `CC=#{ENV.cc}`